src/test/kotlin/com/terraformation/backend/accelerator/db/ModuleStoreTest.kt

```diff
@@ -0,0 +1,1210 @@
+                              endDate = LocalDate.of(2024, 1, 31),
+                              projects = setOf(projectId),
+                          )),
+                  eventDescriptions =
+                      mapOf(
+                          EventType.LiveSession to "Live session lectures",
+                          EventType.OneOnOneSession to "1:1 meetings",
+                          EventType.Workshop to "Workshop ideas",
+                      ),
+                  eventSessions = emptyMap(),
+                  overview = "<h> Overview </h>",
+                  preparationMaterials = "<i> Preps </i>",
+              )),
+          store.fetchAllModules())
+    }
+
+    @Test
+    fun `returns list of modules with multiple modules`() {
+      val moduleId =
+          insertModule(
+              additionalResources = "<b> Additional Resources </b>",
+              liveSessionDescription = "Live session lectures",
+              name = "TestModule",
+              oneOnOneSessionDescription = "1:1 meetings",
+              overview = "<h> Overview </h>",
+              preparationMaterials = "<i> Preps </i>",
+              phase = CohortPhase.Phase1FeasibilityStudy,
+              workshopDescription = "Workshop ideas",
+          )
+
+      val otherModuleId =
+          insertModule(name = "OtherTestModule", phase = CohortPhase.Phase0DueDiligence)
+
+      insertCohortModule(cohortId, moduleId, LocalDate.of(2024, 1, 1), LocalDate.of(2024, 1, 31))
+
+      assertEquals(
+          listOf(
+              ModuleModel(
+                  id = moduleId,
+                  name = "TestModule",
+                  phase = CohortPhase.Phase1FeasibilityStudy,
+                  additionalResources = "<b> Additional Resources </b>",
+                  cohorts =
+                      listOf(
+                          CohortModuleModel(
+                              cohortId = cohortId,
+                              startDate = LocalDate.of(2024, 1, 1),
+                              endDate = LocalDate.of(2024, 1, 31),
+                              projects = setOf(projectId),
+                          )),
+                  eventDescriptions =
+                      mapOf(
+                          EventType.LiveSession to "Live session lectures",
+                          EventType.OneOnOneSession to "1:1 meetings",
+                          EventType.Workshop to "Workshop ideas",
+                      ),
+                  eventSessions = emptyMap(),
+                  overview = "<h> Overview </h>",
+                  preparationMaterials = "<i> Preps </i>",
+              ),
+              ModuleModel(
+                  id = otherModuleId,
+                  name = "OtherTestModule",
+                  phase = CohortPhase.Phase0DueDiligence,
+              ),
+          ),
+          store.fetchAllModules())
+    }
+
+    @Test
+    fun `returns list of modules with multiple modules and multiple cohorts`() {
```

Comment:
This could test the "module is used by more than one cohort" case as well if one of the modules is in both cohorts and the other is only in one cohort.

At which point the test method above this one could probably go away since this covers that functionality too.